test/parallel/test-repl.js

```diff
@@ -414,7 +414,14 @@ function error_test() {
       expect: `${prompt_multiline}'foo \\n'\n${prompt_unix}` },
     // Whitespace is not evaluated.
     { client: client_unix, send: ' \t  \n',
-      expect: prompt_unix }
+      expect: prompt_unix },
+    //Do not parse `...[]` as a REPL keyword
```

Comment:
It passes as we do not use the [spaced-comment](http://eslint.org/docs/rules/spaced-comment) rule. I fixed it anyways.